src/Compilers/Core/Portable/Syntax/SyntaxDiffer.cs

```diff
@@ -65,7 +67,7 @@ private IList<TextChange> ComputeTextChangesFromOld()
             this.ComputeChangeRecords();
             var reducedChanges = this.ReduceChanges(_changes);
 
-            return reducedChanges.Select(c => new TextChange(c.Range.Span, c.NewText)).ToList();
+            return reducedChanges.Select(c => new TextChange(c.Range.Span, c.NewText!)).ToList();
```

Comment:
It looks like `before` and `after` should be marked as nullable on line 73 (below)

Reply:
Also on line 39 (above)